### PR TITLE
headlamp-plugin: Install tar and update dependencies-sync

### PR DESCRIPTION
**Pull request by skoeva**
This change addresses the plugins test failure in the CI.

Fixes: #2324

#### Testing
Tested on issue #1996 in [this commit](https://github.com/headlamp-k8s/headlamp/commit/ddd6c4a3cc49b67878bdfe05af285a37c9c6eb06).
- [x] Run `make plugins-test` and ensure that it passes

### CODE REVIEW

**Comment by vyncent-t** [comment]
it would be good to detail out the reproduction steps, I'm assuming its meant to be ran at the root level of the project.

I also seem to be hitting a different error than the one you have in the original issue

image

**Comment by vyncent-t** [comment]
also, I would try

- pull from main for the most recent changes
- remove the node modules folders in the frontend and plugins
- run npm i for frontend and then npm i for the plugins
- run make plugins-test again

File: frontend/package.json [hunk 1]
```diff
@@ -92,6 +92,7 @@
     "spacetime": "^7.4.0",
     "stream-browserify": "^3.0.0",
     "stream-http": "^3.2.0",
+    "tar": "^7.4.3",
```
Comment (sniok) [inline frontend/package.json:95]: tar is only missing from the plugins/headlamp-plugin, no need to add to the frontend

**Review by sniok — CHANGES_REQUESTED** [review]
nice fix, thanks! just remove tar from frontend and we can merge this

**Review by sniok — APPROVED** [review]

**Pull request merged**